sdk/rust/oak_config_serializer/src/main.rs

```diff
@@ -78,13 +87,45 @@ impl Default for InitialNodeConfig {
     }
 }
 
+fn get_sha256(data: &[u8]) -> String {
+    let mut hasher = Sha256::new();
+    hasher.update(data);
+    hex::encode(hasher.finalize().as_slice().to_vec())
```

Comment:
Not sure if this works

```suggestion
    hex::encode(hasher.finalize().collect())
```

Reply:
Finalize returns a [`GenericArray`](https://docs.rs/generic-array/0.14.2/generic_array/struct.GenericArray.html) and it doesn't have `collect`.